packages/components/package.json

```diff
@@ -72,10 +73,13 @@
     "@talend/bootstrap-theme": "^8.2.0",
     "@talend/design-system": "^7.15.1",
     "@talend/icons": "^6.60.1",
+    "@talend/eslint-plugin": "^1.0.1",
```

Comment:
this is a dependency of scripts-core so you should not need it as it is not used directly